recipes/xapian/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "xapian" %}
+{% set version = "1.4.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://oligarchy.co.uk/xapian/{{ version }}/xapian-bindings-{{ version }}.tar.xz
+  sha256: 4519751376dc5b59893b812495e6004fd80eb4a10970829aede71a35264b4e6a
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - xapian-core={{ version }}
+    - zlib
+
+  host:
+    - python
+    - pip
+    - sphinx  # TODO: this shouldn't be needed, but there's no way to disable it in configure
+
+  run:
+    - python
+    - xapian-core={{ version }}
+
+test:
+  imports:
+    - xapian
+
+about:
+  home: http://xapian.org
+  license: GPL-2.0+
+  license_family: GPL
+  license_file: LICENSE.txt
```

Comment:
It looks like there's a `COPYING` file with the GPL license in the source tarball. In that case, you needn't include license files in this PR. Instead, replace the above line with `about/license_file: COPYING`, and the file should be [copied into the built package info section](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#license-file).

Reply:
Alright, I'll go with that one then.